src/status_im/contexts/chat/group_create/events.cljs

```diff
@@ -30,7 +31,7 @@
  (fn [_ [{:keys [chat-id group-name color image]}]]
    {:json-rpc/call [{:method      "chat_editChat"
                      :params      ["" chat-id group-name (name color)
-                                   {:imagePath image
+                                   {:imagePath (when image (string/replace-first image #"file://" ""))
```

Comment:
could  you please give some details? thanks

Reply:
hi @flexsurfer, Thank you very much for reviewing PR.

Without replacing `file://` rpc call was returning an error
![image](https://github.com/status-im/status-mobile/assets/17097240/aece13d8-71bb-4580-abae-aa11eed844c7)

So I used the same solution as we currently have for profile picture
https://github.com/status-im/status-mobile/blob/df939bf85fd03f1a6a1da0e2ba8d9252949d92e1/src/status_im/contexts/profile/edit/header/events.cljs#L30
(@mohsen-ghafouri probably knows why and how this works)

btw, unless I made some mistake while testing it seems group picture didn't worked for android, even in https://github.com/status-im/status-mobile/pull/19128

Reply:
thank you, i was wondering why we have to do it only here and not as a general solution

Reply:
created and issue: https://github.com/status-im/status-mobile/issues/20454

Reply:
thank you

Reply:
It's not a general issue on all android versions, but by replacing "file://", it converts the URL format to a plain file path, which is more universally accepted by various Android APIs and libraries